src/pages/student/students/StudentDetail/index.js

```diff
@@ -102,8 +104,16 @@ const StudentDetail = () => {
     <center className={style.studentDetailContainer}>
       <div className={style.student}>
         <div className="d-flex">
-          <div className={style.profile}>
-            <BiUser className={style.biUserPosition} />
+          <div>
+            {studentDetails?.avatar ? (
+              <div>
+                <img src={avatar} className={style.biUserPosition} />
+              </div>
+            ) : (
+              <div>
+                <img className={style.biUserPosition1} src="https://www.pngall.com/wp-content/uploads/5/Profile-Avatar-PNG.png"/>
+              </div>
+            )} 
```

Comment:
maybe we can add the ternary in the src and classname instead?
```suggestion
          <div>
            <div>
                <img src={studentDetails?.avatar ? studentDetails?.avatar : 'https://www.pngall.com/wp-content/uploads/5/Profile-Avatar-PNG.png'} className={tudentDetails?.avatar ? style.biUserPosition : style.biUserPosition1} />
              </div>
```